### PR TITLE
Updated sec-1 to match ping example code

### PR DESCRIPTION
**Pull request by raymondsiu**
I noticed the Getting Started section still uses Actix 0.8 and Futures but the ping example has moved on to use the newer async await standard without Futures.

I did run cargo test before committing and there's one outstanding failed test for sec-1:

```
test sec_1_getting_started_sect_ping_actor_line_115 ... FAILED
warning: struct is never constructed: `Ping`
 --> /tmp/rust-skeptic.Is0iZA6jfrsE/test.rs:5:1
  |
5 | struct Ping(usize);
  | ^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(dead_code)]` on by default
```

I'm not entirely sure, but I think this test failed due to skeptic.rs not being able to grok the new await syntax so it thinks the Ping struct wasn't used.

This is my first Rust-related PR so let me know if there's anything I missed! I would greatly appreciated the guidance!

### CODE REVIEW

**Comment by JohnTitor** [comment]
This crate is edition 2015 now so we should make it edition 2018.

**Comment by raymondsiu** [comment]
Where can I specify the 2018 edition?

**Comment by JohnTitor** [comment]
You can specify the edition on `Cargo.toml` on the root.

**Comment by JohnTitor** [comment]
We should update all examples at once to make CI green.

File: Cargo.toml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+cargo-features = ["edition"]
```
Comment (JohnTitor) [inline Cargo.toml:1]: Is it really needed?

**Pull request closed without merging**